web/src/main/java/se/inera/intyg/webcert/web/web/controller/legacyintegration/FragaSvarUthoppController.java

```diff
@@ -115,15 +136,19 @@ public Response redirectToIntyg(@Context UriInfo uriInfo,
         @PathParam("intygId") String intygId,
         @QueryParam("enhet") String enhetHsaId) {
 
-        super.validateParameter("intygId", intygId);
-        super.validateAuthorities();
-        //This is hardwired to fk7263 only
-        String intygType = DEFAULT_TYPE;
-        String intygTypeVersion = DEFAULT_TYPE_VERSION;
-        this.validateAndChangeEnhet(intygId, intygType, enhetHsaId);
-
-        LOG.debug("Redirecting to view intyg {} of type {}", intygId, intygType);
-        return buildRedirectResponse(uriInfo, intygType, intygTypeVersion, intygId);
+        if (reactPilotUtil.useReactClientFristaende(webCertUserService.getUser(), null)) {
```

Comment:
Look at moving this login into "buildRedirectResponse". 

The logic to validate parameters and unit etc should always be executed.

Reply:
Fixed